worker/compile.go

```diff
@@ -2,8 +2,10 @@ package worker
 
 // Compiling anything that's more compicated than single file:
 //
-// - Prepare a "compile_%s.sh" file, with %s being the language (cc, go, rs, java, py2, py3, pas).
-// - Prepare any more files as needed. They will all be put into the CWD of the script.
+// - Prepare a "compile_%s.%ext" file, with %s being the language (cc, go, rs, java, py2, py3, pas) and
+//      TODO: %ext being the shell extension (.sh for linux and .bat/.ps1 for windows).
```

Comment:
This is going to be a massive pain to users... If only some cross-platform scripting language exists ~~not Python~~.
Not sure if we want to fully support both here.

Reply:
How about we use Makefile?

Reply:
Makefiles assume bash either, so I'm not sure how that translates to Windows ;)

Reply:
Checkout this issue https://github.com/natsukagami/kjudge/issues/95 . I think we can implement rudimentary support for "stage" first.

Reply:
Let's revert this comment since it's not actually implemented yet?

Reply:
Yeah. Will do ASAP.